resources/common.rb

```diff
@@ -0,0 +1,34 @@
+actions :install
+default_action :install
+
+# Name the resource
+attribute :name, :name_attribute => true, 'kind_of' => String,
```

Comment:
There is no need to redefine the name attribute, all resources get this one for free.